tests/command_line/test_search_beam_position.py

```diff
@@ -207,28 +205,20 @@ def test_search_small_molecule(dials_data, run_in_tmp_path):
         assert shift.elems == pytest.approx((0.091, -1.11, 0), abs=1e-2)
 
 
-def test_multi_sweep_fixed_rotation(dials_regression, run_in_tmp_path):
-    data_dir = os.path.join(dials_regression, "indexing_test_data", "multi_sweep")
-    reflection_files = sorted(
-        glob.glob(os.path.join(data_dir, "SWEEP[1,2]", "index", "*_strong.pickle"))
-    )
-    experiment_files = sorted(
-        glob.glob(
-            os.path.join(data_dir, "SWEEP[1,2]", "index", "*_datablock_import.json")
-        )
-    )
+def test_multi_sweep_fixed_rotation(dials_data, run_in_tmp_path):
+    data = dials_data("l_cysteine_dials_output", pathlib=True)
+    experiments_path = data / "imported.expt"
+    refl_path = data / "strong.refl"
 
-    search_beam_position.run(reflection_files + experiment_files)
+    search_beam_position.run([os.fspath(experiments_path), os.fspath(refl_path)])
```

Comment:
Question: what is `os.fspath` doing here? I thought you can transparently use `path` objects as a string-like thing?

Reply:
Without that I get
```
TypeError: Please report this error to dials-support@lists.sourceforge.net: 'PosixPath' object is not subscriptable
```
although it probably also works just to call `str` rather than `os.fspath`

Reply:
Here it does nothing and is probably not hurting anything because we _know_ that we get `pathlib.Path` objects from dials_data - in general you might want to accept anything that has `__fspath__`.

That said, above in this PR _is_ explicitly using `str(Path)` so I've changed this to match.